### PR TITLE
Add metadata to Cargo.toml

### PR DESCRIPTION
**Pull request by raphaelrobert**
This PR adds a few more fields to Cargo.toml.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -3,6 +3,11 @@ name = "openmls"
 version = "0.2.0"
 authors = ["Raphael Robert"]
```
Comment (franziskuskiefer) [inline Cargo.toml:4]: Authors should change to something like "The openmls developers <some@email.com>"

File: Cargo.toml [hunk 2]
```diff
@@ -3,6 +3,11 @@ name = "openmls"
...
 edition = "2018"
+description = "This is a PoC Rust implementation of the Messaging Layer Security (MLS) protocol based on draft 9+."
```
Comment (franziskuskiefer) [inline Cargo.toml:6]: I'd not call it PoC but something like WiP or whatever. But it's supposed to be a production library. It's just not there yet.

File: Cargo.toml (hunk 1 above)
Comment (raphaelrobert) [reply Cargo.toml:4]: Agreed, as I'll change it to that as soon as we have a DL set up.

**Review by franziskuskiefer — APPROVED** [review]

File: Cargo.toml (hunk 1 above)
Comment (raphaelrobert) [reply Cargo.toml:4]: I also added that to #196.

**Pull request merged**